packages/components/src/form/index.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import { cloneElement, Component } from '@wordpress/element';
+import { noop } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * A form component to handle form state and provide input helper props.
+ */
+class Form extends Component {
+    constructor( props ) {
+        super();
+
+		this.state = {
```

Comment:
Theres a bit of weird indentation going on in this file, starting with this line here.